### ISSUE TITLE
Toggle: Screen readers cannot read text inside accordion panels

### ISSUE THREAD

**Issue by szet0018** [wet-boew/wet-boew#8918]
https://wet-boew.github.io/v4.0-ci/demos/toggle/toggle-en.html#accordion

In the example above, neither JAWS 2020/NVDA on Chrome 84.0.4147.105 nor NVDA on Firefox 79 can read the text inside the opened panels. Neither ARROW keys nor TAB allows the screen reader to access the text inside the panel when the panel does not have interactive elements like form fields that can be TABed to.

This may be due to the div wrapping around the accordion given role="tablist"
`<div class="accordion" role="tablist">`

role="tablist" automatically triggers application mode in most screen readers.
Regular text in p, headings, div, span, lists, tables, etc. are not available to screen readers in application regions unless it is keyboard focusable.

Tried adding role="document" around the panel, but this did not solve the problem. Not sure what the solution is.

**Comment by RubyDo** [comment]
CC @duboisp  to stay on the loop for the issue

**Comment by GormFrank** [comment]
Relates to #8922

**Comment by piopi** [comment]
@duboisp @GormFrank I think this issue can also be closed because of the improvement done in https://github.com/wet-boew/wet-boew/pull/8948#issuecomment-703368337 , Screen readers can read text inside accordion panels.

**Comment by GormFrank** [comment]
Thank you for your contribution @piopi! @szet0018 Do you want us to close this issue right now or do you want to wait and test when it gets to Canada.ca in a few weeks first?
